packages/contracts/contracts/chugsplash/L2ChugSplashDeployer.sol

```diff
@@ -174,46 +174,6 @@ contract L2ChugSplashDeployer is Ownable {
         );
     }
 
-    /**
-     * Allows the owner to cancel the current active upgrade bundle.
-     */
-    function cancelTransactionBundle()
-        public
-        onlyOwner
-    {
-        require(
-            hasActiveBundle() == true,
-            "ChugSplashDeployer: cannot cancel when there is no active bundle"
-        );
-
-        emit BundleCancelled(
```

Comment:
You should delete the event definition above too.

Reply:
Done!